celements-xwiki-core/pom.xml

```
@@ -332,7 +332,7 @@
     <connection>scm:git:git@github.com:celements/celements-xwiki.git</connection>
     <developerConnection>scm:git:git@github.com:celements/celements-xwiki.git</developerConnection>
     <url>https://github.com/celements/celements-xwiki</url>
-    <tag>HEAD</tag>
+    <tag>celements-xwiki-core-4.0-M1</tag>
```

should this remain at
```suggestion
    <tag>HEAD</tag>
```